docs/account.mdx

```diff
@@ -0,0 +1,630 @@
+used as a consensus address. You cannot use the address of your `secp256k1`
+account, because this signature scheme is not supported on consensus.
+
+Withdrawal transactions are not free of charge and the fee will be deducted
+**from your ParaTime balance**.
+
+You can specify a custom Oasis address which you want to withdraw your ROSE to by appending it to the withdraw command:
+
+```
+$ oasis accounts withdraw 10 oasis1qpxhsf7xnm007csw2acaa7mta2krzpwex5c90qu6
+Unlock your account.
+? Passphrase:
+You are about to sign the following transaction:
+Format: plain
+Method: consensus.Withdraw
+Body:
+  To: webext (oasis1qpxhsf7xnm007csw2acaa7mta2krzpwex5c90qu6)
+  Amount: 10.0 TEST
+Authorized signer(s):
+  1. ArEjDxsPfDvfeLlity4mjGzy8E/nI4umiC8vYQh+eh/c (secp256k1eth)
+     Nonce: 0
+Fee:
+  Amount: 0.0011317 TEST
+  Gas limit: 11317
+  (gas price: 0.0000001 TEST per gas unit)
+
+Network:  testnet
+ParaTime: sapphire
+Account:  eugene
+```
+
+## Delegate and Undelegate stake
+
+To stake your tokens on consensus layer, run `account delegate <amount> <to>`.
+This will delegate the specified amount of tokens to a validator.
+
+```
+$ oasis account delegate 20 oasis1qpkl3vykn9mf4xcq9eevmey4ffrzf0ajtcpvd7sk --account oscar --no-paratime
+Unlock your account.
+? Passphrase:
+You are about to sign the following transaction:
+Method: staking.AddEscrow
+Body:
+  To:     oasis1qpkl3vykn9mf4xcq9eevmey4ffrzf0ajtcpvd7sk
+  Amount: 20.0 TEST
+Nonce:  7
+Fee:
+  Amount: 0.0 TEST
+  Gas limit: 1271
+  (gas price: 0.0 TEST per gas unit)
+
+Network:  testnet
+ParaTime: none (consensus layer)
+Account:  oscar
+```
+
+Once your tokens are staked, they are denoted as *shares* since the number of
+tokens will increase over time due to positive interest rate, while the number
+of shares will remain constant. Also, shares are always interpreted as a whole
+number, whereas the amount of tokens is usually a rational number and may lead
+to rounding errors when managing your delegations. To find out how many shares
+do you poses, run `account show` and look for the `shares` under the active
+delegations section:
+
+```
+$ oasis account show oscar --show-delegations
+Address: oasis1qrec770vrek0a9a5lcrv0zvt22504k68svq7kzve
+Nonce: 28
+
+=== CONSENSUS LAYER (testnet) ===
+  Total: 1006.262158403 TEST
+  Available: 984.669098086 TEST
+
+  Active Delegations from this Account:
+    Total: 21.593060317 TEST
+
+    Delegations:
+      - To:     oasis1qpkl3vykn9mf4xcq9eevmey4ffrzf0ajtcpvd7sk
+        Amount: 21.593060317 TEST (20000000000 shares)
+
+  Allowances for this Account:
+    Total: 0.0000002 TEST
+    Allowances:
+      - Beneficiary: oasis1qpkl3vykn9mf4xcq9eevmey4ffrzf0ajtcpvd7sk
+        Amount:      0.0000002 TEST
+
+
+=== emerald PARATIME ===
+Balances for all denominations:
+  1.0 TEST
+```
+
+To undelegate tokens, use `account undelegate <shares> <from>`. You will need
+to specify the **number of shares to undelegate** and the validator address you
```

Comment:
Should explicitly mention that these are not tokens but shares and explain that the value is derived based on the state of the pool that one delegated to.

Should also mention the debonding period and that during that period the delegated amount may still be subject to slashing.